keras2onnx/common/onnx_ops.py

```diff
@@ -405,7 +405,7 @@ def apply_identity(scope, input_name, output_name, container, operator_name=None
     container.add_node('Identity', input_name, output_name, name=name)
 
 
-def apply_softmax(scope, input_name, output_name, container, operator_name=None, axis=1):
+def apply_softmax(scope, input_name, output_name, container, operator_name=None, axis=-1):
```

Comment:
Are you sure -1 is also supported by ONNX?

Reply:
will do this in conv.py